<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="5" hostname="LTILPUN22001550" failures="1" tests="3" name="Test1" time="52.289" errors="0" timestamp="2022-09-23T10:55:58 IST">
  <testcase classname="TestCases_UI.HBI_HomePageTest" name="verifyProfile" time="5.396"/>
  <testcase classname="TestCases_UI.HBI_LoginPageTest" name="verifyAzadiLogo" time="6.973"/>
  <testcase classname="TestCases_UI.HBI_LoginPageTest" name="verifyMOHFWLogo" time="5.3">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestCases_UI.HBI_LoginPageTest.verifyMOHFWLogo(HBI_LoginPageTest.java:49)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyMOHFWLogo -->
  <testcase classname="TestCases_UI.HBI_LoginPageTest" name="verifymohfwLink" time="0.0">
    <ignored/>
  </testcase> <!-- verifymohfwLink -->
  <testcase classname="TestCases_UI.HBI_LoginPageTest" name="verifyloginToHBIforNodalOfficer" time="0.0">
    <ignored/>
  </testcase> <!-- verifyloginToHBIforNodalOfficer -->
  <testcase classname="Base.TestBase" name="@BeforeSuite setUp" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUp -->
  <testcase classname="Base.TestBase" name="@AfterSuite tearDown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDown -->
  <testcase classname="TestCases_UI.HBI_LoginPageTest" name="verifyLoginToHBIforHP" time="0.0">
    <ignored/>
  </testcase> <!-- verifyLoginToHBIforHP -->
</testsuite> <!-- Test1 -->
